go.mod

```diff
@@ -1026,3 +1030,13 @@ replace github.com/prometheus/procfs => github.com/prometheus/procfs v0.12.0
 
 // Use v0.62.0 of prometheus/common for all dependencies until mongodb_exporter is updated to support 0.63.0
 replace github.com/prometheus/common => github.com/prometheus/common v0.62.0
+
+replace github.com/grafana/beyla/v2 => github.com/ptodev/beyla/v2 v2.0.0-20250507083427-93b2af475c8d
+
+replace github.com/open-telemetry/opentelemetry-collector-contrib => github.com/ptodev/opentelemetry-collector-contrib v0.0.0-20250506154750-58322eed65ce
+
+replace github.com/open-telemetry/opentelemetry-collector-contrib/exporter/kafkaexporter => github.com/ptodev/opentelemetry-collector-contrib/exporter/kafkaexporter v0.0.0-20250506154750-58322eed65ce
+
+replace github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkareceiver => github.com/ptodev/opentelemetry-collector-contrib/receiver/kafkareceiver v0.0.0-20250506154750-58322eed65ce
+
+replace github.com/open-telemetry/opentelemetry-collector-contrib/internal/kafka => github.com/ptodev/opentelemetry-collector-contrib/internal/kafka v0.0.0-20250506154750-58322eed65ce
```

Comment:
I'll open PRs upstream to get those issues resolved.

Reply:
We'll need to wait for a [Beyla](https://github.com/grafana/beyla/pull/1904) and an [OTel](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/39946) PRs to be merged.